morceus-data/latin/stems/nominals/ls.nom

```diff
@@ -217244,6 +217425,9 @@ Vescita_ni^a
 
 vescor		<itype>vesci</itype>
 vescu^lus		<itype>i</itype>	<pos>adj.</pos>
+:le:vesculus
+:aj:vescu^l	us_i masc fem neut 
```

Comment:
I know we had some discussion on but, but I just checked and Gaffiot does say -us, -a, -um for this word - should we use that since it will be more general?

Reply:
changed to -us -a -um